design-tokens/Base.json

```diff
@@ -437,5 +437,52 @@
       "type": "lineHeights",
       "description": "Sets a lineheight of 150%"
     }
+  },
+  "BorderRadius": {
+    "0": {
+      "value": "0px",
+      "type": "borderRadius",
+      "description": "Sets a 0px corner-radius"
+    },
+    "100": {
+      "value": "4px",
+      "type": "borderRadius",
+      "description": "Sets a 4px corner-radius"
+    },
+    "200": {
```

Comment:
Why do you use 100, 200, 300 and no calculation and in spacings PR #98 you use a key named as the multiplier of your basis?